x-pack/plugins/canvas/public/components/loading/loading.tsx

```diff
@@ -25,6 +35,11 @@ export const Loading = ({ animated, text, backgroundColor }) => {
   }
 
   const rgb = hexToRgb(backgroundColor);
+  let color = 'text';
```

Comment:
nit: You could avoid using `let` by wrapping this in a helper function that returns the correct value.